src/mongo/db/mongod_options.cpp

```diff
@@ -1259,6 +1300,23 @@ Status storeMongodOptions(const moe::Environment& params, const std::vector<std:
     if (replSettings.getPretouch())
         log() << "--pretouch " << replSettings.getPretouch();
 
+    // Percona Variables
+    if(params.count("Percona.cursorTimeoutMS")) {
```

Comment:
Please, follow common style rules, one of which is to put space between `if` and `(`